doc/release/1.17.0-notes.rst

```diff
@@ -42,18 +42,21 @@ In the future it will not be possible to switch the writeable flag to ``True``
 from python.
 This deprecation should not affect many users since arrays created in such
 a manner are very rare in practice and only available through the NumPy C-API.
+
 Writing to the result of `numpy.broadcast_array` will warn
 ----------------------------------------------------------
 
 Commonly `numpy.broadcast_array` returns a writeable array with internal
 overlap, making it unsafe to write to. A future version will set the
-``writeable`` flag ``False``, and require users to manually set it ``True`` if
-they are sure that is what they want to do. Writing to it will now emit a
-deprecation warning with instructions to set the ``wrietable`` flag ``True``.
-This creates a confusing situation where even though the ``writeable`` flag is
-``True``, it must be set ``True`` again in order to clear the deprecated
-behaviour. An additional `FutureWarning` will be emitted when accessing the
-``writeable`` flag state to clarify the contradiction.
+``writeable`` flag to ``False``, and require users to manually set it to
+``True`` if they are sure that is what they want to do. From this version,
```

Comment:
From this version -> In this version or perhaps "Until then, " depending on what the goal was.

Reply:
Now, writing to it ...